<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectInfo</class>
 <widget class="QMainWindow" name="ProjectInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Info</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="generalInfoGroupBox">
      <property name="title">
       <string>General information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="2">
        <widget class="QTreeWidget" name="generalInfoTreeWidget">
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="indentation">
          <number>20</number>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>400</number>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>40</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Info</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Project Name</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spatial Reference</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>EPSG Code</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Projection</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Extent</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>top</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>right</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bottom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Extent area in mÂ²</string>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>Cellsize</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>X</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Y</string>
           </property>
          </item>
         </item>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="viewInBrowserPushButton">
         <property name="text">
          <string>WebMap</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="descriptionGroupBox">
      <property name="title">
       <string>Description</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="3">
        <widget class="QPlainTextEdit" name="descriptionPlainTextEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="loadToolButton">
         <property name="toolTip">
          <string>Load description from file...</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="saveToolButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Save edits...</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../Images/SaveEdits.png</normaloff>../Images/SaveEdits.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
